marcel-todo/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,49 @@
+class TasksController < ApplicationController
+	before_action :set_task, only: [:show, :edit, :update, :destroy]
+
+	def index
+		@tasks = Task.all
+	end
+
+	def show
+	end
+
+	def edit
+	end
+
+	def update
+		if @task.update_attributes(task_params)
+			redirect_to root_path
+		end
+	end
+
+	def new
+		@task = Task.new
+	end
+
+	def create
+		@task = Task.new(task_params)
+
+		if @task.save
+			redirect_to root_path
+		else
+			redirect_to :back
+		end
+	end
+
+	def destroy
+		@task.destroy
```

Comment:
still having problems with indentation ;)

you may want to ask austen how to configure sublime.